lms/djangoapps/experiments/flags.py

```diff
@@ -170,12 +185,19 @@ def is_enabled_without_course_context(self):
         return self.is_enabled()
 
     def is_experiment_on(self, course_key=None):
-        return super().is_enabled(course_key)
+        # If no course_key is supplied check the global flag irrespective of courses
+        if course_key is None:
+            return super().is_enabled_without_course_context()
+
+        return super().is_enabled(course_key=course_key)
```

Comment:
This wasn't a keyword arg before, but is now... is that right?

Reply:
It doesn't matter in Python. Keword args can also be passed in as positional args. Just a matter of style. For a single-arg function like this, I personally wouldn't use the kwarg style, but I don't care.

Reply:
Thanks, this clears up some confusion for me